Client/src/Components/Inputs/Autocomplete/index.jsx

```diff
@@ -0,0 +1,104 @@
+import { Autocomplete, Stack, TextField, Typography } from "@mui/material";
+import "./index.css";
+import React from "react";
+import PropTypes from "prop-types";
+import parse from "autosuggest-highlight/parse";
+import match from "autosuggest-highlight/match";
```

Comment:
I'm not sure highlighting the text as the user types is worth adding a dependency and importing another library, small as it may be.

I think filtering out options that don't match the user's text is probably sufficicent.

Reply:
Removed ✅